grepos/app/src/main/java/net/boondockradio/grepos/dao/GithubDao.java

```diff
@@ -0,0 +1,29 @@
+package net.boondockradio.grepos.dao;
+
+import net.boondockradio.grepos.api.GithubApi;
+import net.boondockradio.grepos.dto.Repositories;
+
+import rx.Observable;
+import rx.android.schedulers.AndroidSchedulers;
+import rx.schedulers.Schedulers;
+
+public class GithubDao {
+
+    private GithubApi mGithubApi;
+
+    public GithubDao(GithubApi githubApi) {
+        mGithubApi = githubApi;
+    }
+
+    public Observable<Repositories> getRepositories(String q, String type, int page, int per_page) {
+        return mGithubApi
+                .getRepositories(
+                        q,
+                        type,
+                        page + "",
```

Comment:
This works fine. 👍  You may also use `String.valueOf(someValue);` http://www.techscore.com/blog/2012/11/28/%E6%95%B0%E5%80%A4-%E2%87%94-%E6%96%87%E5%AD%97%E5%88%97%E5%A4%89%E6%8F%9B-java%E7%B7%A8/

I think this is more common than concatenating empty string. 

If you are manipulating `String` values in Android, you can also use `TextUtils` class as well which is a utility class that Android platform provides.

https://developer.android.com/reference/android/text/TextUtils.html